src/components/ManifestViews/Browse/BrowsePager/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import style from './style.module.css'
+import Link from 'components/Shared/Link'
+
+const BrowsePager = ({ parentCategory }) => {
+  const links = [
+    { label: 'All', target: '/browse/' },
+    { label: 'Time', target: '/browse/timeperiods/' },
+    { label: 'Place', target: '/browse/places/' },
+    { label: 'Theme', target: '/browse/themes/' },
+  ]
+
+  return (
+    <div className={style.browsePager}>
+      {
+        links.map(link => {
+          return (
+            <Link
+              to={link.target}
+              key={link.target}
+              activeClassName={style.active}
+              className={activeParent(parentCategory, link.target) ? style.active : ''}
```

Comment:
Seems to be the same as what activeClassName is intended for. Are both of these necessary?

Reply:
`activeClassName` is picked up automatically for path matches by Gatsby Link. The additional className is to keep the active style for sub-collections that belong to the appropriate parent category.